packages/neo-one-server-client/src/npmCheck.ts

```diff
@@ -0,0 +1,90 @@
+import { getEscapedNPMName } from '@neo-one/utils';
+import fetch from 'cross-fetch';
+import * as fs from 'fs-extra';
+
+export interface NPMResult {
+  readonly success: boolean;
+  readonly version: string;
+  readonly date: number;
+}
+const emptyNPMResult: NPMResult = {
+  success: false,
+  version: '',
+  date: 0,
+};
+
+const askNPM = async (pkg: string): Promise<NPMResult> =>
+  new Promise<NPMResult>(async (resolve) => {
```

Comment:
Eliminate this promise

Reply:
How's this look? https://github.com/neo-one-suite/neo-one/commit/139445b41d98d4625c7dc097100843bc19a6a9f0#diff-42f186175b31114082c92b9cf786a793R17